### PR TITLE
Added to the pipeline a default scylla version

### PR DESCRIPTION
**Pull request by fgelcer**
## PR pre-checks (self review)

- [ ] I followed [KISS principle](https://en.wikipedia.org/wiki/KISS_principle) and [best practices](https://docs.google.com/document/d/1jihgOKb5iGRlD8_HQ92O0JbLk1kASUoZT23i_MXFSKI)
- [ ] I gave variables/functions meaningful self-explanatory names
- [ ] I didn't leave commented-out/debugging code
- [ ] I didn't copy-paste code
- [ ] I added the relevant `backport` labels
- [ ] New configuration option are added and documented (in `sdcm/sct_config.py`)
- [ ] I have added tests to cover my changes (Infrastructure only - under `unit-test/` folder)
- [ ] All new and existing unit tests passed (CI)
- [ ] I have updated the Readme/doc folder accordingly (if needed)

### CODE REVIEW

**Comment by fgelcer** [comment]
@roydahan , let's check if it will refrain the job to auto-update the scylla version when being called from centos-rpm job.

File: jenkins-pipelines/manager-sanity.jenkinsfile [hunk 1]
```diff
@@ -7,6 +7,7 @@ longevityPipeline(
     params: params,
 
     backend: 'aws',
+    scylla_version: '3.1.2'
```
Comment (fruch) [inline jenkins-pipelines/manager-sanity.jenkinsfile]: this won't work, `vars/longevityPipeline.groovy` doesn't support reading `scylla_version` from this file

**Comment by fruch** [comment]
@fgelcer still need this PR ? if no, please close it

**Comment by fgelcer** [comment]
it was fixed on the post-action stage of rpm build this missing attribute

**Pull request closed without merging**